vision/spec/quickstart_spec.rb

```diff
@@ -0,0 +1,33 @@
+# Copyright 2016 Google, Inc
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+require "rspec"
+require "google/cloud"
+
+describe "Vision Quickstart" do
+
+  it "label a cat image" do
```

Comment:
This description doesn't flow like a proper sentence.

Maybe `it "performs label detection on sample image file"` ?